tests/test_event_handler.py

```diff
@@ -0,0 +1,22 @@
+from unittest import mock, TestCase
+from src.lib.event_handler import EventHandler
+
+
+class EventHandlerTest(TestCase):
+    @staticmethod
+    def test_all_events_called_on_invoke():
+        my_handler = EventHandler()
+
+        listener_1 = mock.Mock()
```

Comment:
mock.Mock() is just a generator for some function, any function. Usually used to override functionality of a real class with some dummy functionality. Here we only care that a specific function was called